dateutil/relativedelta.py

```diff
@@ -249,7 +249,8 @@ def _fix(self):
 
     @property
     def weeks(self):
-        return self.days // 7
+        weeks = abs(self.days) // 7
```

Comment:
I think replace this with `return int(weeks/7.0)`, that will give the right zero-symmetrical floor-div behavior without the if statement.